src/interfaces/categorie.ts

```diff
@@ -0,0 +1,9 @@
+interface Category {
+    id: number;
+    name: string;
+    image: string;
+    creationAt: string;
+    updatedAt: string;
+}
+
+export default Category;
```

Comment:
better don't use `export default` in future

Reply:
fixed